tilt.py

```diff
@@ -73,8 +73,7 @@ def SetTiltRadioButtons(self):
         self.tiltbutton = []
         
         # Pull out the tilt indices and elevations for display
-        tilt_indices = self.Vradar.value.sweep_start_ray_index['data'][:]
-        elevs = self.Vradar.value.elevation['data'][tilt_indices]
+        elevs = self.Vradar.value.fixed_angle['data'][:]
```

Comment:
I need to look through this a little. When you deal with aircraft radar data, naming conventions can get a little weird (some aircraft radars are rotated 90 degrees from the ground). I don't want to change too much of this, but revisit in the future.